doc/article/technical_article.tex

```diff
@@ -111,48 +108,47 @@
 \begin{abstract}
   The main goal of the present article is to describe the idea, goals and main
-  functionalities of the web based application trckr. We developed trckr for
+  functionalities of the web-based application trckr. We developed trckr for
   % Since we want trckr to be all lowercase, we avoid using it at the start of
   % the sentence
   everyone who works on a project and needs an intuitive and simple web tool to
   accurately track their time spent on different tasks.
   % Hint: Empty line makes a paragraph which is semantically different from the
   % forced \\ newline (and much neater)
 
-  The backend is written in Python with the help of the web application
-  framework Django and the frontend with the Javascript UI framework Vue.js.
+  The back-end is written in Python with the help of the web application
+  framework Django and the front-end with the Javascript UI framework Vue.js.
   Both technologies were new to most team members, but all members have proven
-  themselves self-driven and learning these skills was ultimately a benefit to
+  themselves self-driven, and learning these skills was ultimately a benefit to
   the outcome of the project.
 
   In order to compete with similar tools and web services, trckr focuses on
   performance and usability. To distinguish trckr from the competition, many
   features are planned to manage projects and tasks in a user-friendly manner.
-  This will allow the user to leverage trckr to handle the ever increasing
+  This will allow the user to leverage trckr to handle the ever-increasing
   complexity in project management and task tracking found in large companies.
-  Despite targeting large companies, trckr will remain open source and
-  anyone can contribute, covering cases we might have never dreamed of.
+  Despite targeting large companies, trckr will remain open-source and
+  anyone can contribute.
 \end{abstract}
 
 \clearpage
 
 \section{Introduction}
 Time and task tracking are important activities in many businesses to gain
-insights on the productivity of a team, requiring appropriate tools allowing
-easier and more accurate time tracking. Many processes and methods have been
-developed in the past to cover this need. Unfortunately most of them only
-address a certain need or have been fine tuned to a specific company or team.
-This not only leads to a loss of experience that could have been leveraged by
-other teams but is also not generic enough to adapt to different processes,
-causing other companies to inappropriately adapt to the tool instead of the tool
-adapting to the company.
+insights on the productivity of a team. Appropriate tools allow easier and more
+accurate time tracking. Many processes and methods have been developed in the
+past to cover this need. Unfortunately most of them only address a certain need
+or have been fine-tuned to a specific company or team. This not only leads to a
```

Comment:
i would add a verb "... have been fine-tuned to **fit** a specific company or team." or something like that